src/Nethermind/Nethermind.Synchronization/ParallelSync/SyncMode.cs

```diff
@@ -68,16 +68,12 @@ public enum SyncMode
         /// </summary>
         SnapSync = 2048,
         /// <summary>
-        /// Everything is in a beacon node control. There is no need to do any sync action
-        /// </summary>
-        BeaconControlMode = 4096,
-        /// <summary>
         /// Reverse download of headers from beacon pivot to genesis
         /// </summary> 
-        BeaconHeaders = 8192,
+        BeaconHeaders = 4096,
```

Comment:
BeaconControlMode = BeaconHeaders, is it by plan?

Reply:
there is no BeaconControlMode, we are doing WaitingForBlocks (just waiting from beacon node and not from p2p gossip)